pkg/ctl/cmdutils/filter.go

```diff
@@ -145,8 +145,9 @@ func (f *Filter) doAppendIncludeGlobs(names []string, resource string, globExprs
 }
 
 func (f *Filter) doSetExcludeExistingFilter(names []string, resource string) error {
-	f.excludeNames.Insert(names...)
-	for _, n := range names {
+	uniqueNames := sets.NewString(names...).List()
```

Comment:
If the caller is passing duplicate `names`, I think we should do the deduplication there. We can address this later.

Reply:
Yeah, I wasn't too sure what's best really.